apps/rush-lib/src/schemas/rush-project.schema.json

```diff
@@ -52,6 +52,15 @@
         "type": "string"
       },
       "uniqueItems": true
+    },
+
+    "ignoreChangedFiles": {
+      "type": "array",
+      "description": "A list of file globs under the project root that will be ignored when determining whether this project has changed.",
```

Comment:
Possible improved wording:

> The incremental analyzer can skip Rush commands for projects whose input files have not changed since the last build.  Normally, every Git-tracked file under the project folder is assumed to be an input. Use the `incrementalBuildIgnoredGlobs` setting to ignore specific file paths. The paths are specified as globs relative to the project folder.